index.bs

```diff
@@ -190,6 +191,33 @@ The <dfn attribute for="XRSession">environmentBlendMode</dfn> attribute MUST rep
 
 - A blend mode of <dfn enum-value for="XREnvironmentBlendMode">additive</dfn> MUST be reported if the [=XR Compositor=] is using [=additive environment blending=].
 
+
+
+XRInteractionSpace {#xrinteractionspace-enum}
+----------------------
+
+Sometimes the application will wish to draw UI that the user may interact with. WebXR allows for a variety of form factors, including both handheld phone AR and head-worn AR. For different form factors, the UIs will belong in different spaces to facilitate smooth interaction, for example the UI for handheld phone AR will likely be drawn directly on the screen without projection, but the UI for headworn AR will likely be drawn a small distance from the head so that users may use their controllers to interact with it.
```

Comment:
I think there's some ambiguity about what it means to draw directly on the screen. For example, a stereoscopic display may have separate left/right eye views, so applications would still need to respect the viewport split when drawing to the screen, but for projection purposes it would act as screen-space.

Maybe explicitly highlight that applications still need to respect the views array and viewports for rendering?

Reply:
Will do!

Reply:
Though, for a stereoscopic handheld display I suspect you'd actually want to draw directly to the canvas anyway? You don't want to have multiple versions of the same buttons.

(This is precisely the kind of distinction this API lets us make, that an API based on exposing form factor directly does not)